### PR TITLE
Adding collections - closes #206

### LINKED ISSUE #206: Implement Collections
So that Objects can be added to one or more for synchronization w/ Collections pomegranate.

### PR DESCRIPTION
**Pull request by escowles**

### CODE REVIEW

File: spec/validators/exhibit_id_validator.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe ExhibitIdValidator do
+  subject { described_class.new }
+
+  describe "#validate" do
+    let(:errors) { double("Errors") }
+    before do
+      allow(subject).to receive(:existing_exhibit_ids).and_return(['slug1'])
```
Comment (tpendragon) [inline spec/validators/exhibit_id_validator.rb]: Stubbing the object under test is usually bad joojoo. Can you go a level deeper?

**Comment by tpendragon** [comment]
Can we add "Create a Collection" to the front page? Maybe as the first option?

**Comment by tpendragon** [comment]
"Add to Collection" from the search results view pops up a modal which seems to immediately dissapear

File: spec/validators/exhibit_id_validator.rb (hunk 1 above)
Comment (escowles) [inline spec/validators/exhibit_id_validator.rb]: OK, I'll stub ActiveFedora::SolrService.query instead of the ids method.

Comment (tpendragon) [inline spec/validators/exhibit_id_validator.rb]: If that turns out to be too complicated, maybe a query object to encapsulate it?

**Comment by escowles** [comment]
I've added a button to create a Collection to the home page, stubbed the ActiveFedora::SolrService.query method instead of the ExhibitIdValidator, and suppressed links to the broken "Add to a Collection" modal.

**Comment by tpendragon** [comment]
:+1: Will merge when it passes.

**Comment by tpendragon** [comment]
@escowles Coverage. =( Looks like `collectible`

**Comment by escowles** [comment]
Not sure why the feature test didn't cover that -- I've added a controller test to get it back to :100:

**Pull request merged**